init.lua

```diff
@@ -328,26 +349,38 @@ local function commandHandler(...)
     local args = {...}
     if #args == 1 then
         if args[1] == 'sell' and not loot.Terminate then
-            doSell = true
+            if not mq.TLO.Cursor() then
+                doSell = true
+            else
+                addRule(mq.TLO.Cursor(), mq.TLO.Cursor():sub(1,1), 'Sell')
+                loot.logger.Info(string.format("Setting \ay%s\ax to \ay%s\ax", mq.TLO.Cursor(), 'Sell'))
+            end
         elseif args[1] == 'reload' then
             lootData = {}
-            loot.logger.Info("Reloaded Loot File")
+            loadSettings()
+            loot.Terminate = false
+            loot.logger.Info("Reloaded Settings And Loot Files")
         elseif args[1] == 'bank' then
             loot.bankStuff()
+        elseif args[1] == 'tribute' then
+            if not mq.TLO.Cursor() then
+                doTribute = true
+            else
+                addRule(mq.TLO.Cursor(), mq.TLO.Cursor():sub(1,1), 'Tribute')
+                loot.logger.Info(string.format("Setting \ay%s\ax to \ay%s\ax", mq.TLO.Cursor(), 'Tribute'))
+            end
         elseif args[1] == 'loot' then
             loot.lootMobs()
         elseif args[1] == 'tsbank' then
             loot.markTradeSkillAsBank()
+        elseif validActions[args[1]] and mq.TLO.Cursor() then
+            addRule(mq.TLO.Cursor(), mq.TLO.Cursor():sub(1,1), validActions[args[1]])
+            loot.logger.Info(string.format("Setting \ay%s\ax to \ay%s\ax", mq.TLO.Cursor(), validActions[args[1]]))
         end
     elseif #args == 2 then
```

Comment:
the reason i hadn't just used what was on the cursor initially was that i wanted the ability to add items i didn't have using the commands, like if i saw a link to something and wanted to go add a rule for that item. so my hotkeys in buttonmaster were just `/lootutils keep "${Cursor.Name}"`.
fine with using what's on the cursor if a name isn't provided but would still like to keep the option of providing the name.

Reply:
i can see the merit in that also, and had the same idea. if item on cursor use it if not use arg 2. I will add that back for dual functionality.